omnibus_overrides.rb

```diff
@@ -4,8 +4,6 @@
 #
 # NOTE: You MUST update omnibus-software when adding new versions of
 # software here: bundle exec rake dependencies:update_omnibus_gemfile_lock
```

Comment:
It may be helpful to insert a blank line to keep the comment about train separate from the comment block above

Reply:
I added a commit to this PR removing that comment altogether. It's actually not relevant in this project.